src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static int limitingLine = 100;

```

Comment:
```suggestion
    private static final int limitingLine = 100;
```

limitingLine - bad naming
also, read rules about constant naming